pkg/operator/rbac.go

```diff
@@ -37,7 +37,7 @@ func MakeRBACObjects(name, namespace, component string, additionalRules []rbacv1
 
 	crb := rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: rbacName,
+			Name: fmt.Sprintf("%s-%s", component, name),
```

Comment:
The clusterrolebinding needs a `fluent-operator` prefix too.

Reply:
> The clusterrolebinding needs a `fluent-operator` prefix too.

Agree, we can use code like below?
```suggestion
			Name: fmt.Sprintf("%s-%s", rbacName, name),
```